### PR TITLE
docs(gatsby-image): Add withIEPolyfill documentation

### PR DESCRIPTION
**Pull request by ooloth**
## Description

Adds instructions for using the new `gatsby-image/withIEPolyfill` import, which adds IE support for `object-fit`/`object-position` to `gatsby-image` via the [`object-fit-images`](https://www.npmjs.com/package/object-fit-images) polyfill.

Feel free to reorganize or rephrase as needed. 👍 

## Related Issues

Related to #12681.

### CODE REVIEW

**Comment by ooloth** [comment]
I see there are some Prettier linting errors...

My computer is in the shop for repairs for a few days, so I updated the docs via the GitHub UI (which is why Prettier didn't have a chance to run). 

I'll hopefully have my computer back by the end of the week, but in the meantime please feel free to push improvements!

File: packages/gatsby-image/README.md [hunk 1]
```diff
@@ -127,6 +127,30 @@ export const query = graphql`
...
+// Other imports...
+import Img from "gatsby-image/withIEPolyfill"
+
+export default ({ data }) => (
+  <div>
+    <h1>Hello gatsby-image</h1>
+    <Img 
+      fixed={data.file.childImageSharp.fixed}
+      objectFit="cover"
+      objectPosition="50% 50%"
```
Comment (marcysutton) [inline packages/gatsby-image/README.md]:
```suggestion
      objectPosition="50% 50%"
      alt=""
```
Prompt the user to include an alt attribute, even if empty, for accessibility.

**Comment by ooloth** [comment]
Thanks, @marcysutton!

I noticed the lack of `alt` attributes in the README, but wasn't sure if I should add them or keep the commit focused on the new polyfill. The nudge is appreciated and I'll keep it in mind in the future. 👍

**Comment by marcysutton** [comment]
This looks great, thanks for submitting a PR @ooloth. I'm going to go ahead and merge this, and we can always improve upon it later on!

**Pull request merged**